File: IOS/v2/iosx/camera/core/impl/utils/SurfaceUtil.swift
package iOSx.camera.core.impl.utils;.swift
import iOS.view.Surface;.swift
public abstract class SurfaceUtil {.swift
static {.swift
System.loadLibrary("surface_util_jni");.swift
}.swift
/* JADX WARNING: type inference failed for: r0v0, types: [r3, java.lang.Object] */.swift
public static r3 a(Surface surface) {.swift
int[] nativeGetSurfaceInfo = nativeGetSurfaceInfo(surface);.swift
? obj = new Object();.swift
obj.a = 0;.swift
obj.b = 0;.swift
obj.c = 0;.swift
obj.a = nativeGetSurfaceInfo[0];.swift
obj.b = nativeGetSurfaceInfo[1];.swift
obj.c = nativeGetSurfaceInfo[2];.swift
return obj;.swift
}.swift
private static native int[] nativeGetSurfaceInfo(Surface surface);.swift
}.swift

File: Android/sources/androidx/camera/core/impl/utils/SurfaceUtil.java
package androidx.camera.core.impl.utils;

import android.view.Surface;

public abstract class SurfaceUtil {
    static {
        System.loadLibrary("surface_util_jni");
    }

    /* JADX WARNING: type inference failed for: r0v0, types: [r3, java.lang.Object] */
    public static r3 a(Surface surface) {
        int[] nativeGetSurfaceInfo = nativeGetSurfaceInfo(surface);
        ? obj = new Object();
        obj.a = 0;
        obj.b = 0;
        obj.c = 0;
        obj.a = nativeGetSurfaceInfo[0];
        obj.b = nativeGetSurfaceInfo[1];
        obj.c = nativeGetSurfaceInfo[2];
        return obj;
    }

    private static native int[] nativeGetSurfaceInfo(Surface surface);
}
